src/components/NewFeatureAlertBrowseAndRequest/index.jsx

```diff
@@ -54,11 +54,7 @@ const NewFeatureAlertBrowseAndRequest = ({ enterpriseId, enterpriseSlug, intl })
           })}
         </Button>,
       ]}
-      dismissible={intl.formatMessage({
```

Comment:
[inform] `dismissible` expects a boolean, not a string. Throws a prop type warning during development. The default "Dismiss" CTA text is already translated via Paragon itself.